2019-20/D05X Individual file tests.sps

```diff
@@ -282,6 +283,7 @@ aggregate outfile = Old_Summary
     /Age_mean = mean(age)
     /sex_Males sex_Females = Sum(Male Female)
     /n_NSUs n_Dead = SUM(NSU deceased)
+    /n_preventable_admissions n_preventable_beddays = Sum(preventable_admissions preventable_beddays)
```

Comment:
Could this be `total_preventable_beddays` please

Reply:
Sure! That makes more sense :) Just testing the negative beddays code above then ill copy across years if all looks good to you